lexmapr/pipeline.py

```diff
@@ -966,15 +966,15 @@ def get_gram_chunks(num):
                                 for suff in range(len(suffixes)):
                                     suffixString = suffixes[suff]
                                     sampleRevisedWithSuffix = grm + " " + suffixString
-                                if (sampleRevisedWithSuffix in resource_terms_revised.keys() and not localTrigger):  # Not trigger true is used here -reason
-                                    # resourceId = resourceRevisedTermsDict[sampleRevisedWithSuffix]
-                                    partialMatchedList.append(sampleRevisedWithSuffix)
-                                    status_addendum.append("Suffix Addition- " + suffixString + " to the Input")
-                                    for eachTkn in grmTokens:
-                                        covered_tokens.append(eachTkn)
-                                        if eachTkn in remaining_tokens:
-                                            remaining_tokens.remove(eachTkn)
-                                    localTrigger = True
+                                    if (sampleRevisedWithSuffix in resource_terms_revised.keys() and not localTrigger):  # Not trigger true is used here -reason
+                                        # resourceId = resourceRevisedTermsDict[sampleRevisedWithSuffix]
+                                        partialMatchedList.append(sampleRevisedWithSuffix)
+                                        status_addendum.append("Suffix Addition- " + suffixString + " to the Input")
+                                        for eachTkn in grmTokens:
+                                            covered_tokens.append(eachTkn)
+                                            if eachTkn in remaining_tokens:
+                                                remaining_tokens.remove(eachTkn)
+                                        localTrigger = True
```

Comment:
@lexmapr Can you confirm this was an indentation typo? This if-statement is duplicated 5 times for 1-5 component matching, and it is only indented to the same level of the for-loop in the 2-component matching section.

Changing this indentation effected one test: test_full_term_sfx_match.